common/lib/xmodule/xmodule/course_module.py

```diff
@@ -855,6 +855,14 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    minimum_grade_credit = Float(
```

Comment:
@cahrens The acceptance criteria for this story say that course teams are responsible for setting the minimum grade required for credit.  I'll confirm this with Griff, but if that's the case, I think it makes sense for the minimum grade to be set in Advanced Settings and stored with the course, even if enabling/disabling credit is configured outside of Studio by EdX staff.

I'm thinking of this as being similar to how we configure certificates.  Course teams can use Advanced Settings to configure things like how the organization name is displayed on certificates, but enabling/disabling certificates is something that's configured outside of Studio by EdX staff.